### PR TITLE
Add ERC: Security token interface for confidentiality and legal status transparency to crypto security tokens

### PR DESCRIPTION
**Pull request by itinance**
This PR suggests a Smart contract interface standard to additionally offer confidentiality and legal status transparency to crypto securities according to the German Electronic Security Act ([eWpG](https://www.gesetze-im-internet.de/ewpg/index.html))

### CODE REVIEW

File: ERCS/erc-xxx.md [hunk 1]
```diff
@@ -0,0 +1,321 @@
+---
```
Comment (abcoathup) [inline ERCS/erc-xxx.md:1]:
```suggestion
---
eip: 7570
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.

File: ERCS/erc-xxx.md [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+title: Smart Contract Interface to implement confidentiality and legal status transparency to crypto security tokens
```
Comment (abcoathup) [inline ERCS/erc-xxx.md:2]:
```suggestion
title: Security token interface for confidentiality and legal status transparency
```
Suggest a shorter title

File: ERCS/erc-7570.md [hunk 3]
```diff
@@ -0,0 +1,322 @@
...
+    );
+```
+
+## Backwards Compatibility
+
+This new token standard does not impose any backward compatibility.
+
+## Copyright
+
+Copyright and related rights waived via CC0.
```
Comment (SamWilsn) [inline ERCS/erc-7570.md:322]: Please use the exact waiver from [the template](https://github.com/ethereum/EIPs/blob/master/eip-template.md?plain=1#L120).

**Review by SamWilsn — CHANGES_REQUESTED** [review]
You'll need to address all the issues found by `eipw` as well: https://github.com/ethereum/ERCs/actions/runs/7112458879/job/19362450600

**Pull request closed without merging**